### PR TITLE
Add peek function to Corpus

### PR DESCRIPTION
**Pull request by R9295**

### CODE REVIEW

**Comment by domenukk** [comment]
I think the name should indicate a bit better what we're peeking, like `peek_next_free_id()` or a part of that

**Comment by domenukk** [comment]
Otherwise looks good! :) Btw the repo writes the first character uppercase on commit messages (for some reason)

**Comment by R9295** [comment]
updated. @domenukk

File: libafl/src/corpus/ondisk.rs [hunk 1]
```diff
@@ -106,6 +106,12 @@ where
         self.inner.replace(idx, testcase)
     }
 
+    /// Peek the next free corpus id
+    #[inline]
```
Comment (domenukk) [inline libafl/src/corpus/ondisk.rs:110]: Not sure if that should be `#[inline]` btw

Comment (domenukk) [reply libafl/src/corpus/ondisk.rs:110]: I guess it doesn't matter, we can leave it

**Comment by R9295** [comment]
@domenukk all green!

File: libafl/src/corpus/cached.rs [hunk 2]
```diff
@@ -147,6 +147,12 @@ where
         self.inner.next(idx)
     }
 
+    /// Peek the next free corpus id
+    #[inline]
+    fn peek_next_free_id(&self) -> CorpusId {
```
Comment (domenukk) [inline libafl/src/corpus/cached.rs:152]: just `peek_free_id` is probably precise enough, what do you think? Change it or leave it? @R9295

**Pull request merged**